src/Fantomas/Program.fs

```diff
@@ -271,58 +283,70 @@ let main argv =
 
     let version = results.TryGetResult <@ Arguments.Version @>
 
+    let inline reraiseAsync (e: exn) =
+        let edi = ExceptionDispatchInfo.Capture e
+        edi.Throw()
```

Comment:
`reraise` doesn't work in computation expressions, this is the best work around.

However this is will stop all execution if one fails. I think we discussed that we should try as many files as we can, collect error results at the end. Is that right? @dawedawe @nojaf

Reply:
I would prefer to do as many files as we can, yes.

Reply:
Yup, I think all files should be formatted and at the end, we see which ones failed.

Reply:
Ok this is changed to keep going thru all files. Reporting setup currently to look like:

`[17:42:59 INF] Formatted: 45, Ignored : 0, Unchanged : 72, Errored: 0`